controllers/binding_controller.go

```diff
@@ -76,10 +76,11 @@ type BindingReconciler struct {
 	GetCFServiceKeyCredentials cfservice.KeyGetter
 	GetServiceName             resource.ServiceNameGetter
 	GetServiceRoleCRN          iam.ServiceRolesGetter
-	SetControllerReference     ControllerReferenceSetter
+	SetControllerReference     OwnerReferenceSetter
```

Comment:
Is this still needed here?

Reply:
Yep! We still have controller references set on the Binding's generated secrets 👍